### PR TITLE
Add sensor / range finder information at Setup tab

### PR DESCRIPTION
**Pull request by HThuren**
Use information from firmware 1.46 where MSP_SENSOR_CONFIG now return hardware information regarding sensor / range finder.

shown here, if configured:
image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by HThuren** [comment]
Comma easy to fix, but it seems the setup for sonar has disappeared. I have a HCSR04 unit, but need to figure the setup 
Found this [add sonar to Naze32 FC](https://www.google.com/search?q=connect+sonar+betaflight+configurator&oq=connect+sonar+betaglight&aqs=chrome.2.69i57j33i10i160l3.19202j0j7&sourceid=chrome&ie=UTF-8#fpstate=ive&vld=cid:dd6a1027,vid:2ngA6MSWw80), more complicated since it use Naze32 motor5 and 6 connector and this [Sonar in Betaflight](https://betaflight.com/docs/development/Sonar)

**Comment by HThuren** [comment]
Sensor was a feature at Configuration tab in betaflight 4.2 ...

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
